bears/general/IndentationBear.py

```diff
@@ -12,6 +12,12 @@
 
 class IndentationBear(LocalBear):
 
+    LANGUAGES = {"All"}
```

Comment:
not really all, can it? I'd rather have a generator here that puts the names of the programming languages *actually supported by the coalangs* here. CC @abhsag24